Script/AtomicEditor/ui/playmode/PlayerOutput.ts

```diff
@@ -47,7 +47,8 @@ class PlayerOutput extends Atomic.UIWindow {
         if (text.length > 32768)
             this.output.text = "";
 
-        this.output.appendText(ev.message + "\n");
+        this.output.appendText("\n" + ev.message);//Ensures all subsequent \n operators and player text outputs correctly
```

Comment:
Let's put comments directly above the line they are commenting.  We'll have linting soon which should catch stuff like this...